ispdb/comcast.net.xml

```diff
@@ -9,46 +9,61 @@
       <port>993</port>
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <incomingServer type="imap">
       <hostname>imap.comcast.net</hostname>
       <port>143</port>
       <socketType>STARTTLS</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <incomingServer type="pop3">
       <hostname>mail.comcast.net</hostname>
       <port>995</port>
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <incomingServer type="pop3">
       <hostname>mail.comcast.net</hostname>
       <port>110</port>
       <socketType>STARTTLS</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
       <authentication>password-cleartext</authentication>
     </incomingServer>
     <outgoingServer type="smtp">
       <hostname>smtp.comcast.net</hostname>
       <port>465</port>
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
       <authentication>password-cleartext</authentication>
     </outgoingServer>
     <outgoingServer type="smtp">
       <hostname>smtp.comcast.net</hostname>
       <port>587</port>
       <socketType>STARTTLS</socketType>
       <username>%EMAILADDRESS%</username>
+      <authentication>OAuth2</authentication>
       <authentication>password-cleartext</authentication>
     </outgoingServer>
-    <documentation url="http://customer.xfinity.com/help-and-support/internet/setting-up-thunderbird/">
+    <documentation url="https://www.xfinity.com/support/articles/setting-up-thunderbird">
       <descr lang="en">Instructions for setting up Thunderbird for use with comcast.net for home users.</descr>
       <descr lang="es">Instrucciones para instalar Thunderbird para el uso de Comcast para dueños de casa.</descr>
     </documentation>
+    <documentation url="https://www.xfinity.com/support/articles/third-party-email-access">
+      <descr lang="en">Instructions for enabling IMAP/POP/SMTP access for third-party clients</descr>
+    </documentation>
   </emailProvider>
+  <oAuth2>
+    <issuer>comcast.net</issuer>
+    <scope>https://email.comcast.net/ profile openid</scope>
```

Comment:
Do we need the `openid` scope for IMAP, POP3 and SMTP? Do we need `profile`?

Reply:
This is what they told me, so that's what I've done.

Reply:
Yes, we now know from testing that we need those.